sdk/recoveryservices-siterecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery/src/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj

```diff
@@ -6,14 +6,14 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.RecoveryServices.SiteRecovery</PackageId>
     <Description>Provides developers with libraries for the updated recovery services site recovery feature under Azure Resource manager.</Description>
-    <Version>2.0.8-preview</Version>
+    <Version>3.0.0-preview</Version>
```

Comment:
I believe 2.1.0-preview would be better because there's no breaking change, only feature addition.

Reply:
updated to 2.1.0